packages/playground/website/src/components/log-modal/index.tsx

```diff
@@ -20,7 +20,13 @@ export function LogModal(props: { description?: JSX.Element; title?: string }) {
 	const [logs, setLogs] = useState<string[]>([]);
 	const [searchTerm, setSearchTerm] = useState('');
 
-	useEffect(getLogs, [activeModal]);
+	useEffect(() => {
+		getLogs();
+		logger.addEventListener(logEventType, getLogs);
```

Comment:
@raicem you could use this event in case if you needed to listen for logger changes that are not fatal errors.